week10_planning/seminar_MCTS.ipynb

```diff
@@ -71,6 +71,7 @@
     "import sys, os\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #90.                ,
</pre><p>I suggest clarifying briefly how the 'is_done' variable is calculated, whether from truncated or terminated states.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yandexdataschool/Practical_RL/pull/534/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>